Use GitHub API based sysl modules

ChloePlanet opened the pull request:
Works with
- `go run main.go --serve demo/demo.sysl --redoc --type=html`
- `make docker-mermaid-run`

ChloePlanet committed:
move sizzle.sysl to sysl-examples repo

ChloePlanet committed:
update project

cuminandpaprika commented on Makefile:
```diff
@@ -12,20 +12,19 @@ test:
+	docker run -it -v $$(pwd)/demo/markdown:/out:rw -v $$(pwd)/demo:/usr/demo:ro anzbank/sysl-catalog demo/demo.sysl
 docker-mermaid-run: docker-mermaid
-	docker run -e SYSL_PLANTUML=localhost:8080 -v $$(pwd)/demo/html:/out:rw -v $$(pwd)/demo:/usr/demo:ro sysl-catalog-mermaid demo/sizzle.sysl --redoc --mermaid --type=html -o /out
+	docker run -e SYSL_PLANTUML=localhost:8080 -v $$(pwd)/demo/html:/out:rw -v $$(pwd)/demo:/usr/demo:ro sysl-catalog-mermaid demo/demo.sysl --redoc --mermaid --type=html -o /out
```
We're missing an environment variable here to use github mode for Sysl modules 👍 
```suggestion
	docker run -e SYSL_PLANTUML=localhost:8080 -e SYSL_MODULES=github -v $$(pwd)/demo/html:/out:rw -v $$(pwd)/demo:/usr/demo:ro sysl-catalog-mermaid demo/demo.sysl --redoc --mermaid --type=html -o /out
```

cuminandpaprika commented on Makefile:
```diff
@@ -12,20 +12,19 @@ test:
 	docker build -t sysl-catalog-mermaid -f mermaid.Dockerfile . 
 docker-run: docker
-	docker run -it -v $$(pwd)/demo/markdown:/out:rw -v $$(pwd)/demo:/usr/demo:ro anzbank/sysl-catalog demo/sizzle.sysl
+	docker run -it -v $$(pwd)/demo/markdown:/out:rw -v $$(pwd)/demo:/usr/demo:ro anzbank/sysl-catalog demo/demo.sysl
```
And also here
```suggestion
	docker run -it -e SYSL_MODULES=github -v $$(pwd)/demo/markdown:/out:rw -v $$(pwd)/demo:/usr/demo:ro anzbank/sysl-catalog demo/demo.sysl
```

cuminandpaprika reviewed (commented):
Looks good, still need to get to the bottom of this certificate issue

cuminandpaprika committed:
Jack/fix docker cert (#190)

* Change docker base image to alpine

The previous image chromedp/headless-shell:latest doesn't have a valid certificate, resulting in an x509 error when making a HTTPS call.
Replacing this with a base alpine image and installing chromium on top resolves this certificate issue

* Add SYSL_MODULES env variables to makefile

cuminandpaprika commented on Makefile:
```diff
@@ -12,20 +12,18 @@ test:
 docker-mermaid-run: docker-mermaid
-	docker run -e SYSL_PLANTUML=localhost:8080 -v $$(pwd)/demo/html:/out:rw -v $$(pwd)/demo:/usr/demo:ro sysl-catalog-mermaid demo/sizzle.sysl --redoc --mermaid --type=html -o /out
-	cp demo/mastercard.yaml demo/html/demo/mastercard.yaml
+	docker run -e SYSL_PLANTUML=localhost:8080 -e SYSL_MODULES=github -v $$(pwd)/demo/html:/out:rw -v $$(pwd)/demo:/usr/demo:ro sysl-catalog-mermaid demo/demo.sysl --redoc --mermaid --type=html -o /out
```
```suggestion
	docker run -e SYSL_PLANTUML=localhost:8080 -e SYSL_MODULES=github -v $$(pwd)/demo/html:/out:rw -v $$(pwd)/demo:/usr/demo:ro sysl-catalog-mermaid demo/demo.sysl --redoc --mermaid --type=html -o /out
	cp ~/.sysl/github.com demo/html/github.com
```

ChloePlanet replied:
Fixed by 987eb5d627e2bb0207793bbb7f8efdb77f66ef9d

ChloePlanet committed:
fix redoc cors

ChloePlanet committed:
update redoc test
